src/components/popupTips.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import {View, Text, StyleSheet, TouchableOpacity} from 'react-native';
+import {FontAwesomeIcon} from '@fortawesome/react-native-fontawesome';
+import {faExclamationCircle} from '@fortawesome/free-solid-svg-icons';
+
+interface Props {
+  title?: string;
+  content: string;
+  btnLeftText?: string;
+  btnRtText?: string;
+  children?: React.ReactNode;
+  callback: () => {};
+}
+
+const PopUpTips = (props: Props) => {
+  const {title, content, btnLeftText, btnRtText, callback} = props;
+  return (
+    <View style={styles.popupWrapper}>
```

Comment:
To use view on the very top of the app please use `react-native-modal` instead. It provides native root view and smooth animation